src/templates/partials/stunnel.hbs

```diff
@@ -0,0 +1,35 @@
+# {{output.header}}
+# {{{output.link}}}
+{{#if (minver "5.40" form.serverVersion)}}
```

Comment:
It looks like the `if` and `/if` clauses don't match as if you set the version below 5.40 this produces an empty config.

Reply:
Also, is `5.40` the earliest support for TLS1.3? The release notes for [Version 5.45, 2018.05.21, urgency: MEDIUM](https://www.stunnel.org/NEWS.html) make it look like TLS1.3 shows up in `5.45`. Can you share where the `5.40` number comes from?

Reply:
My intention was not to support a version that was too old.
Considering that Ubuntu 16.04 LTS will reach its lifecycle in April 2021, and the version available in the Ubuntu 16.04 repository is ```5.39```,So i chose not to export anything below ```5.40```.

But since the question was raised, I'll modify the logic, which will allow to select the lower version.